app/domain/authentication/authn_k8s/application_identity.rb

```diff
@@ -22,6 +22,9 @@ module AuthnK8s
     # validation of ValidateApplicationIdentity
     class ApplicationIdentity
 
+      AUTHENTICATION_CONTAINER_NAME_ANNOTATION = "authentication-container-name".freeze
```

Comment:
This is fine, but FYI you can also denote all string literals in the file as frozen by adding:
```ruby 
# frozen_string_literal: true
```

to the top of the file.

Reply:
oh nice, didn't know that. I'll change it and push again.